README.md

```diff
@@ -14,29 +14,33 @@ This library provides functions to test that a DAML application and its bots are
 
 #### Prerequisites
 
-To be able to develop the library you need:
-* sbt
-
-To use the library in a project:
-* download the library from |PUT LOCATION HERE| as a dependency
+To use the library in a project download the library from Maven Central as a dependency. In a Maven project, the following parts need to be present in pom.xml:
+In the `<dependencies> ... </dependencies>` section, one needs to add the library itself.
+```
+    <dependency>
+        <groupId>com.digitalasset</groupId>
+        <artifactId>functest-java_2.12</artifactId>
+        <version>0.1.9999</version>
+        <scope>test</scope>
+    </dependency>
+```
+In the same section, one has to specify, what SDK to use (e.g. 100.13.16 in the example below).
```

Comment:
When someone using bots, they also have these as a dependency already, but worth mentioning. Both of these need to be at least in test scope, but if they use these already in the project, they will in compile scope.
Also describe why one has to add them separately instead of the library pulling in separately.

Reply:
@bartfaitamas what about writing this?
```
In the same section, one has to specify, what SDK to use (e.g. 100.13.16 in the example below). These dependencies are needed by the testing library but are in the provided scope so they are not transitive dependencies, each project has to pull in the appropriate version of them.
```